blocks/text/text.css

```diff
@@ -71,6 +71,31 @@ main .block.text.border {
   margin-bottom: 1rem;
 }
 
+main .block.text.rounded-border {
+  padding: 0 2rem;
+  margin-bottom: 1rem;
+}
+
+main .block.text.blue {
+  color: #007CAD;
+}
+
+main .block.text.blue h4 {
+  color: #007CAD;
+}
+
+main .block.text.rounded-border p {
+  font-size: var(--body-font-size-xl);
+}
+
+main .block.text.block.text.rounded-border  h4 {
```

Comment:
why does `.block.text` repeat twice?

Reply:
you can remove style
main .block.text.rounded-border